### PR TITLE
GitHub: Improve codebase structure

### PR DESCRIPTION
**Pull request by 0x5bfa**
#### 📋 High priority
- [ ] Something
- [ ] Another thing
- [ ] https://github.com/link/to/an/issue

#### ☑️ PR checklist
- [ ] Were these changes approved in an discussion by the project maintainers to prevent from extra work?
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file after seeing if the string wasn't being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

**Review by ioapiset — APPROVED** [review]

File: docs/CONTRIBUTING.md
Comment (Lamparter) [inline docs/CONTRIBUTING.md:1]: Why'd you delete the docs?

File: docs/code-style.md
Comment (Lamparter) [inline docs/code-style.md:1]: Same here

File: docs/translations.md
Comment (Lamparter) [inline docs/translations.md:1]: Same here

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,14 +8,12 @@ on:
     branches:
       - main
     paths-ignore:
-      - 'assets/**'
-      - 'builds/**'
+      - '.github/assets/**'
       - 'docs/**'
```
Comment (Lamparter) [inline .github/workflows/ci.yml:12]:
```suggestion
```
You deleted the docs

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -8,14 +8,12 @@ on:
...
       - '*.md'
   pull_request:
     paths-ignore:
-      - 'assets/**'
-      - 'builds/**'
+      - '.github/assets/**'
       - 'docs/**'
```
Comment (Lamparter) [inline .github/workflows/ci.yml:17]:
```suggestion
```
Same here

File: .github/assets/logos/fluenthub.png
Comment (Lamparter) [inline .github/assets/logos/fluenthub.png:1]: These assets are used on the UWPC website can you change the links on there and add the banner while you're at it

**Review by Lamparter — CHANGES_REQUESTED** [review]

**Pull request merged**